### DISCUSSION TITLE
Add section in documentation on email-safe CSS

### DISCUSSION THREAD

**Discussion by jsodeman** [mjmlio/mjml#2892] [Ideas]
CSS styling is used in many places in the documentation, but there isn't any content warning that not all CSS is supported in email clients.

I think a short section would be useful both as a warning and as a resource. It could reference URLs like:

- https://www.campaignmonitor.com/css/
- https://designmodo.com/html-css-emails/
- https://www.caniemail.com/

**Comment by iRyusa** [comment]
Yep you're right we should maybe point  to Can I Email as it's the most up to date  resource we have
